src/main/java/org/elasticsearch/cloud/aws/blobstore/S3BlobStore.java

```diff
@@ -71,6 +74,18 @@ public S3BlobStore(Settings settings, AmazonS3 client, String bucket, @Nullable
         }
 
         this.numberOfRetries = maxRetries;
+
+        try{
```

Comment:
Have to use a try/catch here. Otherwise registering a repository will fail if the repository doesn't have permission s3:GetAccelerateConfiguration